app/helpers/application_helper.rb

```diff
@@ -206,7 +206,7 @@ def searchable?
 
   def auto_complete_search(name, val, options = {})
     path = options[:path] || send("#{controller_name}_path")
```

Comment:
In order to do this I would have to add an ugly check like `path.include? 'auto_complete'`.  If i don't then we may end up with URLs like `/katello/products/auto_complete_search/auto_complete_search`.  Thoughts?